cli/src/commands/git.rs

```diff
@@ -1076,11 +1033,16 @@ impl From<RejectedBranchUpdateReason> for CommandError {
     }
 }
 
+// TODO: This could become a formatted string or a function that takes a labeled
+// output as a parameter if we add color to these messages.
+type BranchPushLabel = String;
+
 fn classify_branch_update(
     branch_name: &str,
     remote_name: &str,
     targets: LocalAndRemoteRef,
-) -> Result<Option<BranchPushUpdate>, RejectedBranchUpdateReason> {
+    index: &dyn Index,
+) -> Result<Option<(BranchPushUpdate, BranchPushLabel)>, RejectedBranchUpdateReason> {
```

Comment:
Perhaps, this function can return `Forward|Backward|Sideways` enum in place of formatted message? I think we can move some bits to lib/src/refs.rs later.

Reply:
I think in my separate PR, I'll have

```
enum BranchPushLabel {
MoveForward(...), MoveBackward(...), MoveSideways(...),
Add(...), Delete(...)
}
```